src/app/GitUI/CommandsDialogs/SettingsDialog/Pages/ScriptsSettingsPage.cs

```diff
@@ -70,8 +70,8 @@ public partial class ScriptsSettingsPage : SettingsPageWithHeader
 {cDefaultRemoteUrl}
 {cDefaultRemotePathFromUrl}
 
-File(s):
-{SelectedFiles}
+Diff selection:
```

Comment:
Is this better?

Reply:
> Is this better?

It avoids the possibly misleading term "file".